app/Http/Controllers/RequestSupplierUploadController.php

```
@@ -42,36 +42,29 @@ public function get()
             );
-
             $upload = new RequestSupplierUpload();
             $upload->request_supplier_id = $validated['request_supplier_id'];
             $upload->attachment_name = $validated['attachment_name'];
             $upload->file_location = $fileLocation;
-            $upload->save();
-
-            DB::commit();
-
+            return $upload->save();
+        });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'store'. (undefined)

(StaticAccess)